### ISSUE TITLE
Distinguish different eye movement measures in "response_mode"

### ISSUE THREAD

**Issue by christinabergmann** [langcog/metalab-archive#115]
Hi, 

A source of repeated confusion is that we code hand-coded looking data and automated eye tracking the same way in "response_mode" (used to be eye-tracking, is now according to specs.yaml oculomotor [even though the tabular data still use the old coding] -> ???). 
I would suggest eye-tracking for automated measures and eye-movement for hand-coded. 

Any objections? I know it would require some recoding, but that could be done based on "method".

**Comment by mcfrank** [comment]
I agree that it's confusing but I'm not sure that "tracking" and "movement"
convey what you want...

Mike

On Tue, Feb 7, 2017 at 4:31 AM, Christina Bergmann <notifications@github.com

**Comment by christinabergmann** [comment]
Sure, but in general the split might be useful. Wording suggestions welcome. I thought because eye-tracking is so closely linked with automated, it would be good to continue using this term for the respective subset and add a second descriptor. 
I am also curious how oculomotor happened.

**Comment by christinabergmann** [comment]
I just want to bring this issue back up, it would be great to somehow distinguish automatically tracked results from those based on hand-coding. Maybe even in a separate column that is optional.
